From 119af7919d79bab6ca78debf87855710f0ab12b4 Mon Sep 17 00:00:00 2001
From: Franklin S Cooper Jr <fcooper@ti.com>
Date: Fri, 27 Jan 2017 16:40:43 -0600
Subject: [PATCH 039/205] ARM: OMAP2+: Don't force selecting OMAP_GPMC

This commit comes from:
  git://git.ti.com/processor-sdk/processor-sdk-linux.git

OMAP_GPMC requires CONFIG_MEMORY to be enabled. However, by forcing
OMAP_GPMC to be selected dependencies are ignored. This is a problem
since it means OMAP_GPMC is always enabled even when it shouldn't be.

By avoiding this force selection it insures that OMAP_GPMC is only
enabled when its dependencies are met.

Signed-off-by: Franklin S Cooper Jr <fcooper@ti.com>
Signed-off-by: Sekhar Nori <nsekhar@ti.com>
(cherry picked from commit 46cd490b53c6a850eac6cd72bd4e62e09eea5f76)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm/mach-omap2/Kconfig |    1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/mach-omap2/Kconfig b/arch/arm/mach-omap2/Kconfig
index 940173f..214a516 100644
--- a/arch/arm/mach-omap2/Kconfig
+++ b/arch/arm/mach-omap2/Kconfig
@@ -101,7 +101,6 @@ config ARCH_OMAP2PLUS
 	select MEMORY
 	select MFD_SYSCON
 	select OMAP_DM_TIMER
-	select OMAP_GPMC
 	select PINCTRL
 	select SOC_BUS
 	select OMAP_IRQCHIP
-- 
1.7.9.5

